models.py

```diff
@@ -1428,6 +1430,56 @@ class CognitionPersonalAccessToken(Base):
     token = Column(String)
 
 
+class CognitionPersonalAccessTokenEtl(Base):
+    __tablename__ = Tablenames.PERSONAL_ACCESS_TOKEN_ETL.value
+    __table_args__ = {
+        "schema": "cognition",
+        "include_columns": [
+            "id",
+            "created_by",
+            "created_at",
+            "name",
+            "expires_at",
+            "last_used",
+            "token",
+            "scopes",
+        ],
+    }
+    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    created_by = Column(
+        UUID(as_uuid=True),
+        ForeignKey(f"{Tablenames.USER.value}.id", ondelete="SET NULL"),
+        index=True,
+    )
+    created_at = Column(DateTime, default=sql.func.now())
+    name = Column(String)
+    expires_at = Column(DateTime)
+    last_used = Column(DateTime)
+    token = Column(String)
+    scopes = parent_to_child_relationship(
+        Tablenames.PERSONAL_ACCESS_TOKEN_ETL,
+        Tablenames.PERSONAL_ACCESS_TOKEN_SCOPE_ETL,
+    )
+
+
+class PersonalAccessTokenScopeEtl(Base):
+    __tablename__ = Tablenames.PERSONAL_ACCESS_TOKEN_SCOPE_ETL.value
+    __table_args__ = {"schema": "cognition"}
+    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    created_at = Column(DateTime, default=sql.func.now())
+    scope = Column(String, default=TokenScope.READ_WRITE.value)
+    subject = Column(String, default=TokenSubject.PROJECT.value)
+    subject_id = Column(UUID(as_uuid=True))  # project_id or dataset_id
+    token_id = Column(
```

Comment:
afaik the idea was to later also use this table for non etl tokens wich would make this fkey relation problematic